README.md

```diff
@@ -1,2 +1,5 @@
 # research-archive
 An archive of research
+
+A project from OneVariable 
+James Munns & Amanda Majorowicz
```

Comment:
In markdown, items on back-to-back lines like this get treated as one line. If you want there to be a line break, you either need trailing spaces, or duplicate lines. You could do this instead:

```suggestion

James Munns & Amanda Majorowicz
```